### ISSUE TITLE
build broken on linux after libc update

### ISSUE THREAD

**Issue by mattsawyer77** [ohadravid/netstat2-rs#15]
I'm using netstat2 version v0.9.1. When building on linux with libc at 0.2.155, the build succeeds. After running `cargo update`, my libc version was updated from 0.2.155 to 0.2.161. Then the build started failing with

```
❯ cargo build
   ...
   Compiling netstat2 v0.9.1
error[E0609]: no field `state` on type `&libc::tcp_info`
   --> /home/sawyer/.cargo/registry/src/index.crates.io-6f17d22bba15001f/netstat2-0.9.1/src/integrations/linux/netlink_iterator.rs:193:40
    |
193 |             return TcpState::from(tcpi.state);
    |                                        ^^^^^ unknown field
    |
    = note: available fields are: `tcpi_state`, `tcpi_ca_state`, `tcpi_retransmits`, `tcpi_probes`, `tcpi_backoff` ... and 26 others

For more information about this error, try `rustc --explain E0609`.
error: could not compile `netstat2` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
```

I made the same `cargo update` on my macos/arm64 machine which triggered the same libc update, but building still succeeds there.

**Comment by JerrysShan** [comment]
I meet same error

**Comment by ohadravid** [comment]
Fixed in #17
